main/dapps/dapp-templates.md

```diff
@@ -22,7 +22,40 @@ Gives fungible tokens on request. Frontend is raw JavaScript with
 
 Mints NFT (non-fungible token) baseball cards and lets users buy them with their
 fungible tokens from the Fungible Faucet Dapp. Frontend is raw
-JavaScript with [Material Design Components](https://material-components.github.io/material-components-web-catalog/#/).
+JavaScript with [Material Design
+Components](https://material-components.github.io/material-components-web-catalog/#/).
+
+### OTC Desk Dapp
+
+[Github: dapp-otc](https://github.com/Agoric/dapp-otc)
+
+Gives custom quotes to users, in the form of an exercisable call
+option. This dapp has no frontend, but does have a video tutorial to
+walk through the development.
+
+### Oracle Dapp
+
+[Github: dapp-oracle](https://github.com/Agoric/dapp-oracle)
+
+A generic way to interact with oracles such as the Chainlink
+decentralized oracle network. Frontend is raw JavaScript.
+
+### Pegasus Dapp
+
+[Github: dapp-pegasus](https://github.com/Agoric/dapp-pegasus)
+
+Peg/transfer Agoric digital assets to or from remote entities via the
+Agoric Network API. The Network API notably allows smart contracts
```

Comment:
Link "Agoric Network API" to https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/networking.md